flake.nix

```
@@ -190,6 +190,33 @@
           };
         };
 
+        formatter = pkgs.treefmt.withConfig {
+          runtimeInputs = with pkgs; [
+            nixfmt-rfc-style
+            stylish-haskell
+          ];
+
+          settings = {
+            on-unmatched = "info";
+            tree-root-file = "flake.nix";
+
+            formatter = {
+              stylish-haskell = {
+                command = "stylish-haskell";
+                includes = [ "*.hx" ];
+              };
+              nixfmt = {
+                command = "nixfmt";
+                options = [
+                  "--width"
+                  "80"
```

Does nixfmt's arg parsing support `=` syntax?

If so, this might be easier to read, as the option stays on one line:

```suggestion
                  "--width=80"
```